app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Lile < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Attributes
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count + 1)
+  end
+end
```

Comment:
- [OPTIONAL] You're doing well with the `update_post_likes_counter` method in your `Like` model. Here's a suggestion to enhance your code:

Consider using an `after_save` callback to automatically update the `likes_counter` of the associated post whenever a new like is created. Also, to maintain a clean structure, you can make the `update_post_likes_counter` method private, as it's intended for internal use within the model and doesn't need to be accessed externally.

Here's how you can implement these changes:

```ruby
class Like < ApplicationRecord
  # Associations
  belongs_to :author, class_name: 'User'
  belongs_to :post, class_name: 'Post'

  # Callbacks
  after_save :update_post_likes_counter

  private

  def update_post_likes_counter
    post.update(likes_counter: post.likes.count)
  end
end
```

By following these suggestions, your code will be more organized, adhere to best practices, and maintain a clear separation between public and private methods. Keep up the good work! 🚀👏

Reply:
Implemented all the optional changes suggested.